rca/project_styleguide/templates/patterns/pages/researchcentre/research_centre.html

```diff
@@ -85,7 +84,8 @@ <h2 class="section__heading section__heading--primary heading heading--four">
                         <div class="section__content grid">
                             {% include "patterns/organisms/carousel/carousel.html" with carousel=projects modifier='carousel carousel--no-margin' datatag='data-carousel' %}
                             <div class="section__action">
-                                <a class="link link--primary link--link body body--two" href="/research">
+                                <a class="link link--primary link--link body body--two" href="/research/projects/?school-or-centre={{ page.slug }}#results">
```

Comment:
Hardcoded URLs worry me, Is this a URL that we can load using its name?
`/research/projects/`

Reply:
A valid point, there will only be one research section in the site, but I'm also a bit nervous about depending on these slugs, so I've swapped it for a URL field here in https://github.com/torchbox/rca-wagtail-2019/pull/472/commits/435b39400597d43153f950616b5cf6bf5a8b4c76